### PR TITLE
Cleanup by simplifying strings comparsion with JDK7 Objects.equals(). Part 2.

### PR DESCRIPTION
**Pull request by tbw777**
continuation of #5399

### CODE REVIEW

**Comment by mbien** [comment]
tests fail

**Comment by tbw777** [comment]
Fixed

File: enterprise/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/api/restricted/ResourceConfigurator.java [hunk 1]
```diff
@@ -570,8 +570,7 @@ private String isSameDatabaseConnection(JdbcConnectionPool connPool, String data
                 if ((databaseUrl.indexOf(serverPort) != -1)
                         && ((databaseUrl.indexOf(databaseName) != -1)
                         || (databaseUrl.indexOf(sid) != -1))) {
-                    if ((username != null && user.equals(username))
-                            && (password != null && pwd.equals(password))) {
+                    if (user.equals(username) && pwd.equals(password)) {
```
Comment (mbien) [inline enterprise/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/api/restricted/ResourceConfigurator.java:573]: how is this correct?

Comment (tbw777) [reply enterprise/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/api/restricted/ResourceConfigurator.java:573]: its revert

Comment (tbw777) [reply enterprise/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/api/restricted/ResourceConfigurator.java:573]: hmmmm

Comment (mbien) [reply enterprise/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/api/restricted/ResourceConfigurator.java:573]: no its not reverted!

**Pull request closed without merging**